public/app/features/teams/TeamPermissions.tsx

```diff
@@ -21,6 +21,7 @@ const TeamPermissions = (props: TeamPermissionsProps) => {
       title=""
       addPermissionTitle="Add member"
       buttonLabel="Add member"
+      emptyLabel="There are no members"
```

Comment:
note: I think it could also be the case that a user can't see any members of the team if they are not allowed to list other users. In which case this message is slightly misleading, and it would be more accurate to say something like "there are no members or you do not have the permissions to list the current members".

But I think this is an edge case, and shouldn't happen often, so I'd also be fine with the message as it currently is.

Reply:
Updated! PTAL 🙏

Reply:
Awesome, looks good to me! Thanks for updating!